app/views/hiring_staff/organisations/schools/_school_group.html.haml

```diff
@@ -27,9 +27,9 @@
         %strong.strong= t('school_groups.website', organisation_type: organisation_type_basic(current_organisation).humanize)
       %td.govuk-table__cell.change_answer
         - if current_organisation.website.present?
-          = link_to current_organisation.website, current_organisation.website, class: 'govuk-link wordwrap', target: '_blank'
+          = link_to current_organisation.website, current_organisation.website, class: 'govuk-link wordwrap', target: '_blank', rel: 'noopener'
```

Comment:
What would you say to overwriting `link_to` in our app, so that it always has a rel: 'noopener' (so we don't have to rely on remembering to include it)?

Reply:
It's only really relevant for a small subset of our links (those that open in a new tab/window using `target="blank"`). We could have a separate helper for it but I think it's probably not worth it given we just have a few of them.

Reply:
What would be the demerits of doing something like this?

```ruby
def link_to(args, **options)
  if options[:target] == 'blank'
    options.merge({rel: 'noopener'})
  end
  RailsWherever.link_to(args, options)
end
```

Reply:
Good question. In general, I tend to override/monkey patch existing framework functionality (as opposed to adding a new explicit helper) only as a last resort, because it's usually tending to be too implicit. In this particular instance, it wouldn't be a huge change in and of itself, but someone who's worked on a fair few Rails projects might not expect `link_to` to behave this way.

It would also force this attribute even in those cases where we may _not_ want that (not that I can think of any, to be fair), possibly lead to headaches if the signature of `link_to` changes in a future Rails upgrade, and possibly cause issues if a library that we use overrides `link_to` as well. You could also argue that if this really was behaviour any link should always adopt, it would already be part of Rails core.

Reply:
The `safe_target_blank` gem takes a slightly different approach by overriding `UrlHelper`'s `convert_options_to_data_attributes`:
https://github.com/jvenezia/safe_target_blank/blob/master/lib/safe_target_blank/url_helper.rb

But I feel like adding an extra library for the sake of saving ourselves a handful of characters in the code is probably overkill 😅  Plus we now have Brakeman to keep a watchful eye out for us!

Reply:
Thanks!